server/controllers/content.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable sort-keys */
+/* eslint-disable consistent-return */
+const Content = require("../models/Content");
+
+// Create and Save a new Content
+exports.create = (req, res) => {
+
+    // Validate request
```

Comment:
Go through each of your files and use [http-status-code](https://www.npmjs.com/package/http-status-codes) module to define your status code and the error message associate with each code